README.md

```diff
@@ -45,7 +45,19 @@ The `ntripping` utility has the following usage:
             --url <url>             [default: na.skylark.swiftnav.com:2101/CRS]
 
 Different resources can be requested from different locations. By default, a San
-Francisco latitude, longitude, and height will be used.
+Francisco latitude, longitude, and height will be used. The data returned is in RTCM format.
+
+The `--url` must be formatted as such: `{username}:{passsword}@{area}.skylark.swiftnav.com:{port}/{mountpoint}`, 
+where `username` and `password` refer to the user's [device registry](https://device-registry.cs.swiftnav.com/#/)
+username and password. 
```

Comment:
```suggestion
The `--url` must be formatted as such: `{username}:{passsword}@{area}.skylark.swiftnav.com:{port}/{mountpoint}`, 
where `username` and `password` refer to the user's NTRIP username and password -- for Swift Skylark users
this can be looked up from the Device Registry (internal to Swift).
```